tla/consistency/TraceMultiNodeReads.tla

```diff
@@ -0,0 +1,137 @@
+-------------------------------- MODULE TraceMultiNodeReads -------------------------------
+EXTENDS MultiNodeReads, Json, IOUtils, Sequences
+
+\* Trace validation has been designed for TLC running in default model-checking
+\* mode, i.e., breadth-first search.
+ASSUME TLCGet("config").mode = "bfs"
+
+\* Note the extra /../ necessary to run in the VSCode extension but not a happy CLI default
+JsonFile ==
+    IF "JSON" \in DOMAIN IOEnv THEN IOEnv.JSON ELSE "../../build/consistency/trace.ndjson"
+
+JsonLog ==
```

Comment:
This will become quite obvious once we start matching the tx values in each line, which I can't quite do until #6129, but is very much the next step that follows.